src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -324,20 +324,13 @@ public void initialize() {
    * scoring
    */
   public void cancelScore() {
-    // means startScore has run
-    if (desiredPosition == LiftPosition.PRE_SCORE_HIGH_CONE
-        || desiredPosition == LiftPosition.PRE_SCORE_MID_CONE) {
-      // but finishScore hasn't run
-      if (desiredGrabberClosed) {
-        // check to see if startScore had changed the desiredPosition to HIGH or MID
-        if (desiredPosition == LiftPosition.PRE_SCORE_HIGH_CONE) {
-          setDesiredPosition(LiftPosition.SCORE_HIGH_CONE);
-        } else {
-          setDesiredPosition(LiftPosition.SCORE_MID_CONE);
-        }
-        setCancelScore(true);
-      }
-    }
+    setCancelScore(true);
```

Comment:
Let's add some logic where this only does anything between StartScore and FinishScore? a boolean `scoringInProgress` that StartScore immediately sets to true, then finish score immediately sets back to false? And `finishScoreCancelled` would set it to false as well